public/index.html

```diff
@@ -43,7 +43,7 @@
       .force("center", d3.forceCenter(width / 2, height / 2));
 
   d3.json("/graphdata.json", function(error, graph) {
-    if (error) throw error;
+    if (error) return;
```

Comment:
As per the new comment on L96, this error should be logged anyway - even with a bare generic description such as `No graph data available at "/graphdata.json"`.
All errors must be logged away from the UI, in the developer console if related to the frontend and in the CLI if related to the backend.